Source/WebCore/platform/graphics/avfoundation/objc/ImageDecoderAVFObjC.mm

```diff
@@ -432,15 +438,22 @@ SampleFlags flags() const override
         return false;
     }
 
-    if (m_imageRotationSession)
+    if (m_imageRotationSession) {
         pixelBuffer = m_imageRotationSession->rotate(pixelBuffer.get());
```

Comment:
If we are switching to the rotated version, should we do something to clear the Ownership Identity in the original pixel buffer?

Reply:
No, it still exists due to the requester requesting it. You can think of the size of the frame being 2x what you'd expect since it needs orientation change.
In this case the original will be destroyed as the ref goes out of scope, but until the destruction is real, the attribution (pedantically speaking) assigns the blame to the correct owner.